ibm5150, ibm5170_cdrom: add various FreeDOS versions

davide125 opened the pull request.

DopefishJustin commented:
Also, please create parent-clone relationships between minor revisions or variations.

davide125 commented:
Updated, thanks @DopefishJustin. I left `freedos13rc4_720` in `ibm5150.xml` (as that uses 720k disks) and moved the rest over to ibm5170. I've also added parent/clone between format variants.
